### DISCUSSION TITLE
What permissions are required for a user to change her/his password?

### DISCUSSION THREAD

**Discussion by cupcakearmy** [directus/directus#5911] [Archive]
If i create a user wit the default role permissions s*he can't chante the password for the account. What permissions am I missing?
Note: I don't have smtp setup so recover password email link is not an option.

Below are the default permissions for that role if it helps :)

<img width="803" alt="Screenshot 2021-05-27 at 10 15 49" src="https://user-images.githubusercontent.com/14948823/119790875-9362c500-bed4-11eb-983f-2bf3f87b22c7.png">

**Comment by nazariydj** [comment]
Hi, you need to add Custom Update permissions to Directus Users collection. 
Set Filter Rule in Item Permissions tab like this: 
```
{
    "id": {
        "_eq": "$CURRENT_USER"
    }
} 
```
and select fields user can change (password in your case) in Field Permissions tab.

[Permissions](https://docs.directus.io/guides/permissions/#update-custom-access)
[Filter Rules](https://docs.directus.io/reference/filter-rules/#examples)

**Reply by cupcakearmy** [reply]
Thanks! Worked like a charm.
Might be sensible to add as the reccomended defaults, as most users should be able to update their password
